news/2023/2023-08-22-osu-malaysia-animangaki.md

```diff
@@ -0,0 +1,32 @@
+---
+layout: post
+title: "osu! Malaysia @ AniManGaki 2023"
+date: 2023-08-22 08:00:00 +0000
+---
+
+The osu! Malaysia community proudly presents its first community-run space happening at [AniManGaki 2023](https://www.animangaki.com/)!
+
+![](/wiki/shared/news/2023-08-22-osu-malaysia-animangaki/banner.jpg)
+
+The event will be happening at the Mines International Exhibition & Convention Centre Malaysia (*MIECC*) from 25th to 27th August 2023, providing an opportunity for osu! players all around the country to meet up and mingle!
```

Comment:
https://miecc.mines.com.my/about/

.. although whether "mines" is capitalised seems inconsistent between sources <.<

```suggestion
The event will be happening at the MINES International Exhibition & Convention Centre (*MIECC*) in Malaysia from 25 to 27 August 2023, providing an opportunity for osu! players all around the country to meet up and mingle!
```